testnets/autheotestnet/chain.json

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "autheotestnet",
+  "status": "live",
+  "website": "https://autheo.com/",
+  "network_type": "testnet",
+  "chain_type": "cosmos",
+  "pretty_name": "Autheo",
+  "chain_id": "autheo_785-1",
+  "bech32_prefix": "aauth",
+  "daemon_name": "autheod",
+  "node_home": "$HOME/.autheod",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "aauth",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0.025,
+        "high_gas_price": 0.04
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "aauth"
+      }
+    ],
+    "lock_duration": {
+      "time": "1209600s"
+    }
+  },
+  "codebase": {
+    "git_repo": "https://code.zeeve.net/client-projects/autheo",
+    "genesis": {
+      "name": "v3",
+      "genesis_url": "https://code.zeeve.net/client-projects/autheo/-/blob/autheo-node1/autheo-node1/config/genesis.json"
+    },
+    "recommended_version": "v25.0.0",
```

Comment:
I suspect this may not be the current version, but just copy and pasted from the README. Also, the version must be defined in a 'versions.json' file.